solver/test/compute_order.hpp

```diff
@@ -112,11 +113,67 @@ order ()
   return a;
 }
 
+template <typename Algorithm_t, typename T=double>
+T
+order_lv (Algorithm_t algo=Algorithm_t(1e-5))
```

Comment:
I'm not sure English people can understand this name... 
Could you rename it as `check_high_order` or something similar?

Reply:
Is `lv` a commonly used abbreviation for Lokta-Voltera? If yes, it's good for me.

Reply:
I rename `order` into `short_time_check_order` and `order_lv` into `long_time_check_order` and add a function `check_order` for all methods with `if constexpr`.